### PR TITLE
BY 3 unported licenses

### PR DESCRIPTION
**Pull request by dpoirier**
## Description
Add the BY* 3.0 unported licenses

## Checklist
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the *default* branch of the repository (`main` or `master`).
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [x] I added tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: licenses/management/commands/load_html_files.py [hunk 1]
```diff
@@ -227,26 +226,13 @@ def handle(self, input_directory, **options):
...
-                            message_key = (
-                                "If You Share Adapted Material You produce, the Adapter's "
-                                "License You apply must not prevent recipients of the Adapted "
-                                "Material from complying with this Public License."
-                            )
-                        else:
-                            message_key = english_by_license_code[license_code][
-                                internal_key
-                            ]
+                        message_key = english_messages[internal_key]
```
Comment (Audiosutras) [inline licenses/management/commands/load_html_files.py:235]: `KeyError: 'license_long'`

Comment (Audiosutras) [reply licenses/management/commands/load_html_files.py:235]: This key error seems to only be generated if I run `load_html_files` without the `--versions` argument.

Comment (Audiosutras) [reply licenses/management/commands/load_html_files.py:235]:
```
python3.8 manage.py load_html_files --versions=1.0,3.0 ../creativecommons.org/docroot/legalcode/ 
```
Works as expected. I think the above error is probably tied to 4.0 licenses.
```
python3.8 manage.py load_html_files --versions=4.0 ../creativecommons.org/docroot/legalcode/
```
If I run 4.0 licenses by themselves; this error is generated again.
`KeyError: 's3a4_if_you_share_adapted_material'`

File: cc_licenses/templates/includes/legalcode_30_unported_license.html [hunk 2]
```diff
@@ -0,0 +1,308 @@
+{% load license_tags %}
```
Comment (Audiosutras) [inline cc_licenses/templates/includes/legalcode_30_unported_license.html:1]: I think this template looks good. We just need to update the `side-menu` to have anchor tags to the legal code section headers.

File: licenses/management/commands/load_html_files.py (hunk 1 above)
Comment (dpoirier) [reply licenses/management/commands/load_html_files.py:235]: I'll revert this change. After that, ```load_html_files ../creativecommons.org/docroot/legalcode/``` runs without error for me.

File: cc_licenses/templates/includes/legalcode_30_unported_license.html (hunk 2 above)
Comment (dpoirier) [reply cc_licenses/templates/includes/legalcode_30_unported_license.html:1]: Thanks, I keep forgetting that!

**Comment by dpoirier** [comment]
@Audiosutras This is updated and ready to look at again.

**Review by Audiosutras — APPROVED** [review]
👍

**Pull request merged**